README.md

```diff
@@ -6,60 +6,42 @@ docker/cli
 This repository is the home of the cli used in the Docker CE and
 Docker EE products.
 
-It's composed of 3 main folders
-
-* `/cli` - all the commands code.
-* `/cmd/docker` - the entrypoint of the cli, aka the main.
-
 Development
 ===========
 
-### Build locally
+`docker/cli` is developed using Docker.
 
-```
-$ make build
-```
+Build a linux binary:
 
 ```
-$ make clean
+$ make -f docker.Makefile binary
```

Comment:
Yes, I understand that. 

My concern is that `docker.Makefile` is not really a good use of make. Every target is effectively `.PHONY`.  It is not really taking advantage of any of what Make provides. It could just as easily be a bash script, or ideally a tool that is actually designed for this use case. 

We should be looking to remove `docker.Makefile` and replace it with something else, which is why it's named this way. We could very easily drop in a `dobi.yaml` for this purpose and it would provide a lot of additional benefits.

It should also (eventually) be merged with what is now `circle.yml` so we don't have a separate copy for CI tasks. This is not really possible with circleci, just yet.

We could, for now, create a script that wraps this `make -f docker.Makefile` so the instructions would be something like `./task binary` `./task lint`, etc. This would give us a shim to transition to new build tools and would probably be a lot more concise when what we have in `docker.Makefile` right now.

Since this PR provides some necessary support for building cross, I'd like to get this merged first and address that in a separate PR.

WDTY?